docs/sql-keywords.md

```diff
@@ -103,6 +103,7 @@ Below is a list of all the keywords in Spark SQL.
   <tr><td>DROP</td><td>non-reserved</td><td>non-reserved</td><td>reserved</td></tr>
   <tr><td>ELSE</td><td>reserved</td><td>non-reserved</td><td>reserved</td></tr>
   <tr><td>END</td><td>reserved</td><td>non-reserved</td><td>reserved</td></tr>
+  <tr><td>ESCAPE</td><td>non-reserved</td><td>non-reserved</td><td>reserved</td></tr>
```

Comment:
Also, update `TableIdentifierParserSuite`.

Reply:
Thanks for your reminder. @maropu

Reply:
Ur, IMHO this should be reserved even in spark.

Reply:
OK. I put escape as reserved in spark.